marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -110,4 +110,38 @@ void getSavedQuery() throws Exception {
                     assertEquals(savedQueryResponse.get("savedQuery").get("id").asText(), response.get("savedQuery").get("id").asText());
                 });
     }
+
+    @Test
+    void deleteSavedQuery() throws Exception {
```

Comment:
I'm of the opinion that it's perfectly fine just to toss the test of deletion onto an existing test method, like the existing createThenUpdate test. Fewer test methods means less time setting up tests, which means our test suite is faster.